app/client/main.go

```diff
@@ -10,6 +10,13 @@ import (
 	"github.com/pokt-network/pocket/app/client/cli"
 )
 
+// Provisioned by ldflags
+var (
```

Comment:
Ditto what we have here: https://github.com/pokt-network/pocket/pull/390#discussion_r1046630784

Reply:
https://github.com/pokt-network/pocket/pull/390#discussion_r1053544155